sdks/python/apache_beam/io/parquetio.py

```diff
@@ -506,6 +509,11 @@ def __init__(
         compression_type=CompressionTypes.UNCOMPRESSED)
     self._schema = schema
     self._codec = codec
+    if ARROW_MAJOR_VERSION == 1 and self._codec.lower() == "lz4":
```

Comment:
Technically a str, so:
```suggestion
    if ARROW_MAJOR_VERSION == '1' and self._codec.lower() == "lz4":
```

Reply:
I convert the version string with `int` above